backend/app/api/endpoints/adapter/tasks.py

```
@@ -496,3 +501,156 @@ async def export_task_docx(
+
+    # App data is stored under status.app
+    status_data = task.json.get("status", {}) if task.json else {}
+    app_data = status_data.get("app", {}) if status_data else {}
+    return {"app": app_data}
+
+
+@router.post("/{task_id}/services", response_model=ServiceResponse)
+def update_task_services(
+    service_update: ServiceUpdate,
+    task_id: int = Depends(with_task_telemetry),
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.11)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)